README.rst

```diff
@@ -194,6 +194,21 @@ is enabled or not.
 It is disabled by default because `OpenWISP <http://openwisp.org>`_ does not use
 this feature of `django-organizations <https://github.com/bennylope/django-organizations>`_ yet.
 
+Multitenant mixins
+------------
+
+These are mixins which make a ModelAdmin class multitenant: users will see only the objects related to the
+organizations they are associated with.
```

Comment:
I know you're just copying text, but it's a good chance to improve the docs as well, pelase remove this line because is redundant with what comes next

Reply:
this was not addressed